.devcontainer/docker-compose.yml

```diff
@@ -1,14 +1,28 @@
 name: 'nextcloud-with-memories'
 services:
-  nextcloud:
+  nextcloud-server:
     build:
       context: .
       dockerfile: Dockerfile
     volumes:
+      # - nextcloud:/var/www/html
+      # - apps:/var/www/html/custom_apps
       - ./custom_apps:/var/www/html/custom_apps
-      - ../:/var/www/html/custom_apps/memories
+      - ./../:/var/www/html/custom_apps/memories
       - ./config:/var/www/html/config
+      # - config:/var/www/html/config
       - ./data:/var/www/html/data
+      # - theme:/var/www/html/themes/<YOUR_CUSTOM_THEME>
     ports:
       - 9080:80
-    image: nextcloud
+# volumes:
+# nextcloud:
+#   name: nextcloud
+# apps:
+#   name: apps
+# config:
+#   name: config
+# data:
+#   name: data
+# theme:
+#   name: theme

```

Comment:
Why all these comments?

Reply:
Mostly for other developers that are new to nextcloud (like me) to know that they can mount those folders if needed. Should I remove them?

Reply:
Don't worry about it, I already picked this change. Turns out we need to use volumes anyway for it work right; folders don't play nice with ownership somehow (the server runs as www-data).

The dev container on master should start and set up a full nextcloud instance with memories now (thanks!)